### DISCUSSION TITLE
Is there a way to delete rules from a saved Discover list?

### DISCUSSION THREAD

**Discussion by metaltermite** [jurialmunkey/plugin.video.themoviedb.helper#1287] [Q&A]
I have accidentally used the `Air Date Before` rule, and want to delete it from the saved Discover list. I am unsure how to do this through the UI. Is there a way to do it?

I'm willing to dig around the addon's addon_data files if that's the only way, but I suspect the relevant files are not in readable text format :/

**Comment by jurialmunkey** [comment]
You can edit the saved entries by using Edit from context menu opened on the saved folder. However, while editing it looks like there is a bug where it doesn't prompt to clear the rule if it is already set and setting it to nothing keeps the original value (I'll need to fix that)

Assuming it isn't something massively complicated in terms of the ruleset, it is probably easier to just delete your saved one using context menu and make a fresh one to save with the same name.

**Reply by metaltermite** [reply]
Yeah, from the context menu I can `Edit`, `Rename`, or `Delete` a saved Discover list.

But the specific rules created within it, no. Accessing the context menu from a rule only offers `Add/Remove as Custom Node` and `Add to favourites`. It would definitely be convenient to add a `Remove rule` option here.

Re-creating complicated lists are a pita, mainly due to all the keyword exclusions required.

**Reply by urs45** [reply]
If you add your created rule as a custom node, the saved rules/lists are in the nodes folder: userdata/addon_data/plugin.video.themoviedb.helper/nodes and can be edited with any text editor,
maybe this helps?
